src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -66,20 +66,30 @@ public class DynamoPreferencesData
         public string RunType { get; internal set; }
         public string RunPeriod { get; internal set; }
 
+        /// <summary>
+        /// PolyCurve normal and direction behavior has been made predictable in Dynamo 3.0 and has therefore changed. 
+        /// This reflects whether legacy (pre-3.0) PolyCurve behavior is selected either in preference settings or in the workspace.
+        /// A workspace setting if exists, overrides the default preference setting. 
+        /// </summary>
+        [Obsolete("This property is deprecated and will be removed in a future version of Dynamo")]
+        public bool EnableLegacyPolyCurveBehavior { get; internal set; }
+
```

Comment:
We need public API at the model level ?

Reply:
It needs to be public for serialization purposes.

Reply:
Do we need it to be serialized here?
Is it not enough to have the flag in PreferenceSettings ?

Reply:
If it is per graph, then it is fine as is

Reply:
This is serialized to a sub-section of the DYN json file, which is stored in a separate class, `DynamoPreferencesData`. I tried making it internal but then the DYN just doesn't update.

Reply:
On second thoughts, I suppose this doesn't make sense in the view block, in which case, I need to either move this property to a different class or reuse the property on the workspace itself. Will try that.